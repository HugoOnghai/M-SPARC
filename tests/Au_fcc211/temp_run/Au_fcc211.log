
                            < M A T L A B (R) >
                  Copyright 1984-2019 The MathWorks, Inc.
                  R2019a (9.6.0.1072779) 64-bit (glnxa64)
                               March 8, 2019

 
To get started, type doc.
For product information, visit www.mathworks.com.
 

 Reading .inpt file ...


<INPT>
# $ cat Au_fcc211.inpt
# nprocs: 24
# Test: Au #
LATVEC:
0.0 0.5 0.5
0.5 0.0 0.5
0.5 0.5 0.0
CELL: 15.4 7.7 7.7
MESH_SPACING: 0.2
FD_ORDER: 12
BC: P P P
EXCHANGE_CORRELATION: LDA_PZ
ELEC_TEMP_TYPE: fermi-dirac
ELEC_TEMP: 3000
TOL_SCF: 1e-6
TOL_POISSON: 1e-8
TOL_PSEUDOCHARGE: 1e-8
#ELEC_TEMP: 
CALC_PRES: 1
CALC_STRESS: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
MIXING_VARIABLE: density
MIXING_PRECOND: kerker



<\INPT>

 Reading .ion file ...
 Number of atom types : 1
 Total number of atoms: 2
 Default atomic mass for Au is 196.966569
 pseudo_dir # 1 = Au.psp8

 COORD:
                   0   0.231000000000000   0.077000000000000
   7.391999999999999   0.770000000000000   0.616000000000000

  13.272469414484000

 kpoint grid before symmetry:
     0     0     0

 kpoint grid after symmetry:
     0     0     0

## Chebyshev polynomial degree not provided, finding npl ...
## Based on the mesh size, npl is set to: 35
## Number of states not provided, finding Nev ...
## Based on the number of electrons, Nev is set to: 28
 Creating differentiation matrices ...
 Done. (24.535 sec)

 Estimated memory usage:
 Total:  427.11 MB
 orbitals            :  100.08 MB
 sparse matrices     :  302.01 MB
 global-size vectors :   12.51 MB
 mixing histories    :   12.51 MB

 Finding rb for Au ...
 rb = {3.397 3.356 3.356}, int_b = -19.000702277321778, err_rb = 7.023e-04
 rb = {3.595 3.554 3.554}, int_b = -19.000564148016803, err_rb = 5.641e-04
 rb = {3.792 3.751 3.751}, int_b = -19.000466600083648, err_rb = 4.666e-04
 rb = {3.990 3.949 3.949}, int_b = -19.000397406236839, err_rb = 3.974e-04
 rb = {4.187 4.146 4.146}, int_b = -19.000342032393135, err_rb = 3.420e-04
 rb = {4.385 4.344 4.344}, int_b = -19.000302491791771, err_rb = 3.025e-04
 rb = {4.582 4.541 4.541}, int_b = -19.000267802039367, err_rb = 2.678e-04
 rb = {4.779 4.738 4.738}, int_b = -19.000236584466005, err_rb = 2.366e-04
 rb = {4.977 4.936 4.936}, int_b = -19.000215009008144, err_rb = 2.150e-04
 rb = {5.174 5.133 5.133}, int_b = -19.000197883164176, err_rb = 1.979e-04
 rb = {5.372 5.331 5.331}, int_b = -19.000178466097775, err_rb = 1.785e-04
 rb = {5.569 5.528 5.528}, int_b = -19.000168708577558, err_rb = 1.687e-04
 rb = {5.767 5.726 5.726}, int_b = -19.000158263300236, err_rb = 1.583e-04
 rb = {5.964 5.923 5.923}, int_b = -19.000162328619943, err_rb = 1.623e-04
 rb = {6.162 6.121 6.121}, int_b = -19.000149867027673, err_rb = 1.499e-04
 rb = {6.359 6.318 6.318}, int_b = -19.000139588720202, err_rb = 1.396e-04
 rb = {6.556 6.515 6.515}, int_b = -19.000126047490468, err_rb = 1.260e-04
 rb = {6.754 6.713 6.713}, int_b = -19.000115823747041, err_rb = 1.158e-04
 rb = {6.951 6.910 6.910}, int_b = -19.000106579144461, err_rb = 1.066e-04
 rb = {7.149 7.108 7.108}, int_b = -19.000107161354784, err_rb = 1.072e-04
 rb = {7.346 7.305 7.305}, int_b = -19.000026037777928, err_rb = 2.604e-05
 rb = {7.544 7.503 7.503}, int_b = -18.999997295057042, err_rb = 2.705e-06
 rb = {7.741 7.700 7.700}, int_b = -19.000000585390449, err_rb = 5.854e-07
 rb = {7.938 7.897 7.897}, int_b = -18.999999864051016, err_rb = 1.359e-07
 rb = {8.136 8.095 8.095}, int_b = -19.000000022873515, err_rb = 2.287e-08
 rb = {8.333 8.292 8.292}, int_b = -18.999999998140463, err_rb = 1.860e-09
 rb = {8.333 8.292 8.292}

 ###############################################################
 Relaxation step number: 1 

 Starting pseudocharge generation and self energy calculation...
 Integration b = 37.999999994060

 Integration b_ref = 38.000000000050

 ****************************************
 *          Eself_ref = 1290.382477       *
 ****************************************
 Done. (3.762353 s)
 Time for b calculation: 3.777 seconds.

 Starting calculating nonlocal projectors ...
 Done. (2.257113 s)

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Starting SCF iteration...
 AAR converged to a relative residual of 7.60436e-09 in 67 iterations.
 Poisson problem took 16.121343s
 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 1 
 ============================================= 
 Fermi energy = 5.893837
 This SCF iteration took 19.461 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 2 
 ============================================= 
 Fermi energy = 0.731170
 This SCF iteration took 16.351 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 3 
 ============================================= 
 Fermi energy = 0.089050
 This SCF iteration took 15.558 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 4 
 ============================================= 
 Fermi energy = -0.027271
 Etot = -274.80999424
 Eatom = -137.40499712
 Error in SCF iteration: 2.9834e-02 
 AAR converged to a relative residual of 2.90688e-05 in 19 iterations.
 AAR converged to a relative residual of 7.0225e-09 in 49 iterations.
 Poisson problem took 11.957487s
 This SCF iteration took 32.028 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  2 
 ========================= 
 Fermi energy = -0.027819
 Etot = -274.89244171
 Eatom = -137.44622086
 Error in SCF iteration: 2.0806e-02 
 AAR converged to a relative residual of 3.51291e-05 in 21 iterations.
 AAR converged to a relative residual of 7.8107e-09 in 49 iterations.
 Poisson problem took 11.167607s
 This SCF iteration took 33.439 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  3 
 ========================= 
 Fermi energy = -0.019228
 Etot = -274.89112308
 Eatom = -137.44556154
 Error in SCF iteration: 8.7827e-03 
 AAR converged to a relative residual of 6.23027e-06 in 25 iterations.
 AAR converged to a relative residual of 5.43277e-09 in 49 iterations.
 Poisson problem took 11.927712s
 This SCF iteration took 34.017 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  4 
 ========================= 
 Fermi energy = -0.016989
 Etot = -274.89033418
 Eatom = -137.44516709
 Error in SCF iteration: 4.3350e-03 
 AAR converged to a relative residual of 4.61687e-06 in 25 iterations.
 AAR converged to a relative residual of 6.45664e-09 in 43 iterations.
 Poisson problem took 9.735114s
 This SCF iteration took 33.305 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  5 
 ========================= 
 Fermi energy = -0.017064
 Etot = -274.89000681
 Eatom = -137.44500341
 Error in SCF iteration: 1.7786e-03 
 AAR converged to a relative residual of 6.03749e-06 in 25 iterations.
 AAR converged to a relative residual of 3.10841e-09 in 49 iterations.
 Poisson problem took 11.965658s
 This SCF iteration took 35.163 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  6 
 ========================= 
 Fermi energy = -0.016331
 Etot = -274.88996492
 Eatom = -137.44498246
 Error in SCF iteration: 8.0697e-04 
 AAR converged to a relative residual of 5.69761e-06 in 25 iterations.
 AAR converged to a relative residual of 6.96583e-09 in 43 iterations.
 Poisson problem took 10.533440s
 This SCF iteration took 32.703 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  7 
 ========================= 
 Fermi energy = -0.016467
 Etot = -274.88996475
 Eatom = -137.44498237
 Error in SCF iteration: 3.8533e-04 
 AAR converged to a relative residual of 5.76945e-06 in 25 iterations.
 AAR converged to a relative residual of 9.24819e-09 in 38 iterations.
 Poisson problem took 9.072498s
 This SCF iteration took 32.438 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  8 
 ========================= 
 Fermi energy = -0.016534
 Etot = -274.88996698
 Eatom = -137.44498349
 Error in SCF iteration: 1.5469e-04 
 AAR converged to a relative residual of 5.66634e-06 in 25 iterations.
 AAR converged to a relative residual of 9.65915e-09 in 36 iterations.
 Poisson problem took 7.136445s
 This SCF iteration took 28.305 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  9 
 ========================= 
 Fermi energy = -0.016477
 Etot = -274.88996745
 Eatom = -137.44498372
 Error in SCF iteration: 7.3993e-05 
 AAR converged to a relative residual of 7.52832e-06 in 25 iterations.
 AAR converged to a relative residual of 3.74403e-09 in 37 iterations.
 Poisson problem took 8.665323s
 This SCF iteration took 28.659 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 10 
 ========================= 
 Fermi energy = -0.016477
 Etot = -274.88996772
 Eatom = -137.44498386
 Error in SCF iteration: 2.8333e-05 
 AAR converged to a relative residual of 5.22907e-06 in 25 iterations.
 AAR converged to a relative residual of 4.83189e-09 in 31 iterations.
 Poisson problem took 7.390226s
 This SCF iteration took 30.799 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 11 
 ========================= 
 Fermi energy = -0.016478
 Etot = -274.88996777
 Eatom = -137.44498389
 Error in SCF iteration: 1.7563e-05 
 AAR converged to a relative residual of 4.82609e-06 in 25 iterations.
 AAR converged to a relative residual of 6.16673e-09 in 31 iterations.
 Poisson problem took 6.120397s
 This SCF iteration took 27.635 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 12 
 ========================= 
 Fermi energy = -0.016473
 Etot = -274.88996769
 Eatom = -137.44498385
 Error in SCF iteration: 6.2191e-06 
 AAR converged to a relative residual of 4.67887e-06 in 25 iterations.
 AAR converged to a relative residual of 6.87653e-09 in 25 iterations.
 Poisson problem took 5.814598s
 This SCF iteration took 26.501 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 13 
 ========================= 
 Fermi energy = -0.016473
 Etot = -274.88996771
 Eatom = -137.44498385
 Error in SCF iteration: 4.4243e-06 
 AAR converged to a relative residual of 8.01919e-06 in 25 iterations.
 AAR converged to a relative residual of 7.54097e-09 in 26 iterations.
 Poisson problem took 6.026027s
 This SCF iteration took 28.969 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 14 
 ========================= 
 Fermi energy = -0.016472
 Etot = -274.88996775
 Eatom = -137.44498387
 Error in SCF iteration: 1.3563e-06 
 AAR converged to a relative residual of 3.63979e-05 in 20 iterations.
 AAR converged to a relative residual of 2.23722e-09 in 25 iterations.
 Poisson problem took 5.332565s
 This SCF iteration took 24.840 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 15 
 ========================= 
 Fermi energy = -0.016472
 Etot = -274.88996774
 Eatom = -137.44498387
 Error in SCF iteration: 7.7931e-07 
 This SCF iteration took 15.804 s.


 Finished SCF iteration in 15 steps!
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 

 **********************************************************
 *          Energy per unit cell = -274.889967741 Ha.       *
 *          Energy per atom = -137.444983871 Ha.            *
 **********************************************************

 Starting atomic force calculation ... 
 local force calculation: 4.800 s
 ***********************************************************
 *                      Atomic Force                       *
 ***********************************************************
 Drift free forces (Ha/Bohr):
  -0.005250718462552   0.001576610042152   0.001565089217164
   0.005250718462552  -0.001576610042152  -0.001565089217164

 Max magnitude of forces (Ha/Bohr):   0.005701337374308


 Time for calculating forces: 5.744120 s.

["Stress in GPa"


]  14.748335608140843   2.260221952311944   2.260002445916034
   2.260221952311944  15.073008468109785  -2.313674503092136
   2.260002445916034  -2.313674503092136  15.073537339966023


 Time for calculating stress: 16.036009 s.

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Final atomic positions (Cartesian) are as follows:
  0.000000 	  0.231000 	  0.077000 
  7.392000 	  0.770000 	  0.616000 

 Run-time of the program: 567.562585 seconds

ans = 

  struct with fields:

                  cell_typ: 2
                   lat_vec: [3x3 double]
                  metric_T: [3x3 double]
                    grad_T: [3x3 double]
                    lapc_T: [3x3 double]
                      Jacb: 0.707106781186547
                        L1: 15.400000000000000
                        L2: 7.700000000000000
                        L3: 7.700000000000000
                        Nx: 77
                        Ny: 39
                        Nz: 39
                         N: 117117
                      ecut: -1
              mesh_spacing: 0.200000000000000
                   kptgrid: [0 0 0]
                  kptshift: [0 0 0]
                      nkpt: [1 1 1]
                     tnkpt: 1
                      wkpt: 1
                        BC: 2
                       BCx: 0
                       BCy: 0
                       BCz: 0
                      isBS: 0
                   lattice: 'undefined'
                   SCF_tol: 1.000000000000000e-06
                       Nev: 28
               poisson_tol: 1.000000000000000e-08
          pseudocharge_tol: 1.000000000000000e-08
                       Cst: 27.211386245987999
              Ha_Bohr3_GPa: 2.942101569650548e+04
                        kB: 3.166811563455600e-06
               elec_T_type: 0
                      Temp: 3000
                       bet: 1.052583416013558e+02
                       npl: 35
                       FDn: 6
                    rc_ref: 0.500000000000000
              max_relax_it: 100
                dbg_switch: 0
                        xc: 1
                 Nelectron: 38
                     n_typ: 1
                TimeRevSym: 1
                  spin_typ: 0
                     nspin: 1
              CheFSI_Optmz: 0
          chefsibound_flag: 0
               FixRandSeed: 0
                rhoTrigger: 4
                 NetCharge: 0
                 MAXIT_SCF: 100
                 MINIT_SCF: 3
             MAXIT_POISSON: 1000
            accuracy_level: -1
     target_force_accuracy: -1
    target_energy_accuracy: -1
                 TOL_RELAX: 5.000000000000000e-04
               TOL_LANCZOS: 0.010000000000000
               precond_tol: 3.931480355860088e-05
               precond_kTF: 1
            precond_thresh: 0.250000000000000
            precond_fitpow: 2
          precond_resta_q0: 1.360000000000000
          precond_resta_Rs: 2.760000000000000
            MixingVariable: 0
             MixingPrecond: 1
                  Pf_guess: [117117x1 double]
             MixingHistory: 7
           MixingParameter: 0.300000000000000
     MixingParameterSimple: 0.300000000000000
            PulayFrequency: 1
          PulayRestartFreq: 0
                    TWtime: 1.000000000000000e+09
                 RelaxFlag: 0
                 RelaxMeth: 'LBFGS'
            max_dilatation: 1.200000000000000
            TOL_RELAX_CELL: 0.010000000000000
                    MDFlag: 0
               RestartFlag: 0
                    MDMeth: 'NVE'
                     MD_dt: 1
                  MD_Nstep: 0
                     ion_T: -1
                thermos_TF: -1
              ion_elec_eqT: 1
              ion_vel_dstr: 2
                NLCG_sigma: 0.500000000000000
                     qmass: 1
                 L_history: 20
               L_finit_stp: 0.005000000000000
                  L_maxmov: 0.200000000000000
               L_autoscale: 1
                 L_lineopt: 1
                   L_icurv: 1
                   FIRE_dt: 1
                 FIRE_mass: 1
               FIRE_maxmov: 0.200000000000000
               Calc_stress: 1
                 Calc_pres: 1
            PrintForceFlag: 1
          PrintAtomPosFlag: 1
          PrintAtomVelFlag: 1
         PrintElecDensFlag: 0
            PrintEigenFlag: 0
                PrintMDout: 1
             PrintRelaxout: 1
              Printrestart: 1
           Printrestart_fq: 1
                       xin: 0
                       yin: 0
                       zin: 0
                     twist: 0
                  filename: 'Au_fcc211'
            inputfile_path: '.'
                        XC: 'LDA_PZ'
                       Atm: [1x1 struct]
                     n_atm: 2
                     Atoms: [2x3 double]
           mvAtmConstraint: [2x3 double]
                    IsFrac: 1
                    IsSpin: 0
            is_default_psd: 0
                  temp_tol: 1.000000000000000e-12
                 xc_rhotol: 1.000000000000000e-14
                Atoms_init: [2x3 double]
                  lat_uvec: [3x3 double]
                        dx: 0.200000000000000
                        dy: 0.197435897435897
                        dz: 0.197435897435897
                        dV: 0.005512736496588
                        w2: [1x7 double]
                        w1: [1x7 double]
                         W: [117117x1 double]
            precondcoeff_a: 1
    precondcoeff_lambda_TF: 1
            precondcoeff_k: 0
                    occfac: 2
                Relax_iter: 1
              elecgs_Count: 1
               StressCount: 1
                    amu2au: 1.822888485332371e+03
                    fs2atu: 41.341373336493000
                 isOutl_11: [1001x1 logical]
                 isOutr_11: [1001x1 logical]
                      I_11: [1001x1 double]
                     II_11: [1001x1 double]
                      V_11: [1001x1 double]
                 isOutl_22: [507x1 logical]
                 isOutr_22: [507x1 logical]
                      I_22: [507x1 double]
                     II_22: [507x1 double]
                      V_22: [507x1 double]
                 isOutl_33: [507x1 logical]
                 isOutr_33: [507x1 logical]
                      I_33: [507x1 double]
                     II_33: [507x1 double]
                      V_33: [507x1 double]
                  isOutl_1: [924x1 logical]
                  isOutr_1: [924x1 logical]
                       I_1: [924x1 double]
                      II_1: [924x1 double]
                       V_1: [924x1 double]
                  isOutl_2: [468x1 logical]
                  isOutr_2: [468x1 logical]
                       I_2: [468x1 double]
                      II_2: [468x1 double]
                       V_2: [468x1 double]
                  isOutl_3: [468x1 logical]
                  isOutr_3: [468x1 logical]
                       I_3: [468x1 double]
                      II_3: [468x1 double]
                       V_3: [468x1 double]
                 G_JOutl_1: [1405404x1 logical]
                 G_JOutr_1: [1405404x1 logical]
                     G_J_1: [1405404x1 double]
                     G_I_1: [1405404x1 double]
                     G_V_1: [1405404x1 double]
                 G_JOutl_2: [1405404x1 logical]
                 G_JOutr_2: [1405404x1 logical]
                     G_J_2: [1405404x1 double]
                     G_I_2: [1405404x1 double]
                     G_V_2: [1405404x1 double]
                 G_JOutl_3: [1405404x1 logical]
                 G_JOutr_3: [1405404x1 logical]
                     G_J_3: [1405404x1 double]
                     G_I_3: [1405404x1 double]
                     G_V_3: [1405404x1 double]
                   Lap_std: [117117x117117 double]
                    grad_1: [117117x117117 double]
                    grad_2: [117117x117117 double]
                    grad_3: [117117x117117 double]
                LapPreconL: [117117x117117 double]
                LapPreconU: [117117x117117 double]
              memory_usage: 447855416
               staticfname: 'Au_fcc211.static'
                 suffixNum: 0
                  outfname: 'Au_fcc211.out'
                  parallel: 0
                         b: [117117x1 double]
                     b_ref: [117117x1 double]
                    rho_at: [117117x1 double]
                     Eself: 3.760668177725212e+02
                 Eself_ref: 1.290382477300794e+03
                       V_c: [117117x1 double]
                 PosCharge: 37.999999994060197
                 NegCharge: -37.999999994060197
                    E_corr: -2.309370756847784e-07
                       rho: [117117x1 double]
                      Atom: [1x2 struct]
                       phi: [117117x1 double]
                       Vxc: [117117x1 double]
                 dvxcdgrho: [117117x1 double]
                      Veff: [117117x1 double]
                         X: [117117x7 double]
                         F: [117117x7 double]
          mixing_hist_fkm1: [117117x1 double]
          mixing_hist_xkm1: [117117x1 double]
                       psi: [117117x28 double]
    upper_bound_guess_vecs: [117117x1 double]
                    EigVal: [28x1 double]
                  lambda_f: -0.016471620689090
                       occ: [28x1 double]
                    Etotal: -2.748899677411370e+02
                     Eband: -40.909950429740150
                       Exc: -24.789477181473224
                    Exc_dc: -32.543485249587540
                  Eelec_dc: 1.343486305822402e+02
                      Eent: -0.015837958293156
                   S_Debug: [1x1 struct]
                     force: [2x3 double]
                 abs_force: [2x1 double]
                    Stress: [3x3 double]
                  Pressure: -14.964960472072216
             stress_dimred: [3x3 double]

