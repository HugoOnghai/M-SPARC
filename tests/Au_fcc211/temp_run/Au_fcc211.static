***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of atoms:
      0.0000000000       0.0300000000       0.0100000000
      0.4800000000       0.1000000000       0.0800000000
Atomic forces (Ha/Bohr):
     -0.0052507185       0.0015766100       0.0015650892
      0.0052507185      -0.0015766100      -0.0015650892
Stress (GPa):
  1.4748335608E+01   2.2602219523E+00   2.2600024459E+00 
  2.2602219523E+00   1.5073008468E+01  -2.3136745031E+00 
  2.2600024459E+00  -2.3136745031E+00   1.5073537340E+01 
