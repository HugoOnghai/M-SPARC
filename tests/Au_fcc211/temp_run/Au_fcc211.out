***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 11-May-2021  Start time: 21:09:36                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 15.400000 7.700000 7.700000 
LATVEC:
0.000000000000000 0.500000000000000 0.500000000000000 
0.500000000000000 0.000000000000000 0.500000000000000 
0.500000000000000 0.500000000000000 0.000000000000000 
FD_GRID: 77 39 39
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.009500435
CHEB_DEGREE: 35
NSTATES: 28
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 3.93E-05
PRECOND_KERKER_KTF: 1.00
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: Au_fcc211.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.200000 (Bohr)
Mesh spacing in y-direction        :  0.197436 (Bohr)
Mesh spacing in z direction        :  0.197436 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  Au_fcc211.out
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  38
Atom type 1  (valence electrons)   :  Au 19
Pseudopotential                    :  Au.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  8.33 8.29 8.29
Number of atoms of type 1          :  2
Estimated total memory usage       :  427.11 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -1.3740499712E+02        2.983E-02        83.395
2            -1.3744622086E+02        2.081E-02        33.437
3            -1.3744556154E+02        8.783E-03        34.015
4            -1.3744516709E+02        4.335E-03        33.302
5            -1.3744500341E+02        1.779E-03        35.160
6            -1.3744498246E+02        8.070E-04        32.700
7            -1.3744498237E+02        3.853E-04        32.436
8            -1.3744498349E+02        1.547E-04        28.304
9            -1.3744498372E+02        7.399E-05        28.656
10           -1.3744498386E+02        2.833E-05        30.796
11           -1.3744498389E+02        1.756E-05        27.634
12           -1.3744498385E+02        6.219E-06        26.498
13           -1.3744498385E+02        4.424E-06        28.967
14           -1.3744498387E+02        1.356E-06        24.837
15           -1.3744498387E+02        7.793E-07        15.802
Total number of SCF: 15    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -1.3744498387E+02 (Ha/atom)
Total free energy                  : -2.7488996774E+02 (Ha)
Band structure energy              : -4.0909950430E+01 (Ha)
Exchange correlation energy        : -2.4789477181E+01 (Ha)
Self and correction energy         : -3.7606681800E+02 (Ha)
Entropy*kb*T                       : -1.5837958293E-02 (Ha)
Fermi level                        : -1.6471620689E-02 (Ha)
Average force                      :  5.7013373743E-03 (Ha/Bohr)
Maximum force                      :  5.7013373743E-03 (Ha/Bohr)
Time for force calculation         :  5.744 (sec)
Pressure                           : -1.4964960472E+01 (GPa)
Max stress                         :  1.5073537340E+01 (GPa)
Time for stress calculation        :  16.039 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  567.563 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
